src/DebuggerIdGenerator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Yiisoft\Yii\Debug;
+
+class DebuggerIdGenerator
+{
+    private string $id;
+
+    public function __construct()
+    {
+        $this->generateId();
+    }
+
+    public function getId(): string
+    {
+        return $this->id;
+    }
+
+    public function reset(): void
```

Comment:
Why is this needed?

Reply:
This is for reset id when the debugger restarts in the Roadrunner environment.